src/sentry/static/sentry/app/views/organizationIntegrations/integrationListView/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Control, {OrganizationIntegrations} from './control';
+import Test from './test';
+
+export default function IntegrationListView(props) {
+  if (localStorage.getItem('USE_INTEGRATION_DIRECTORY') === '1') {
```

Comment:
@NisanthanNanthakumar Since we are continuing to send analytics events, we should make sure we set `integration_directory_active` correctly. I think we should move this logic into a function in `integrationUtil.tsx` and then use it to set `integration_directory_active` in `trackIntegrationEvent`.